README.md

```
@@ -131,4 +139,4 @@ To learn more about Organized and how to use it, check out the following useful
 | [Our blog](https://guide.organized-app.com/blog)                                          | Blog with insightful articles filled with tips, tricks, and the latest app update news.                            |
 | [Translation project](https://crowdin.com/project/organized)                              | Collaborate on translating the Organized app – it's easy and convenient.                                           |
 | [Contribution guide](https://github.com/sws2apps/organized-app/blob/main/CONTRIBUTING.md) | See how you can contribute to the Organized project.                                                               |
-| [Telegram channel](https://t.me/+OHxW4AxGTyAxN2Zk)                                        | Get all the latest announcements, release notes, and other useful info instantly in your Telegram messenger.    
\ No newline at end of file
+| [Telegram channel](https://t.me/+OHxW4AxGTyAxN2Zk)                                        | Get all the latest announcements, release notes, and other useful info instantly in your Telegram messenger.    
```

Markdownlint

Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe
Table pipe style

(MD055, table-pipe-style)